.github/workflows/formulaCI.yml

```diff
@@ -0,0 +1,86 @@
+name: Formula CI
+# Description: "Moving saphanabootstrap-formula from Travis to Github Actions"
+# The CI automatically publishes new releases every time a pull request is merged into master
+on: [push, pull_request]
+env:
+  PACKAGE_NAME: saphanabootstrap-formula 
+
+jobs:
+
+  validation:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Add salt repository
+      uses: myci-actions/add-deb-repo@4
+      with:
+        repo: deb http://repo.saltstack.com/apt/ubuntu/18.04/amd64/2019.2/ bionic main
```

Comment:
Can we take advantage here and use the latest salt version?
Here an example: https://github.com/SUSE/habootstrap-formula/blob/master/.github/workflows/formula-ci.yml#L23

Reply:
Yes we can  :D 
I have updated the static version to the latest 
        repo: deb http://repo.saltstack.com/apt/ubuntu/18.04/amd64/latest/ bionic main
        key-server: http://repo.saltstack.com/apt/ubuntu/18.04/amd64/latest/SALTSTACK-GPG-KEY.pub